test/controllers/changeset_controller_test.rb

```diff
@@ -1578,11 +1578,11 @@ def test_query
     assert_response :success, "can't get changesets by closed-ness"
     assert_changesets [3, 5, 6, 7, 8, 9]
 
-    get :query, :closed => "true", :user => users(:normal_user).id
+    get :query, :closed => "true", :user => users(:normal_user)
```

Comment:
Good catch - I'd avoided changing a bunch of other ones, but these two were a mistake.